hyperstart.go

```diff
@@ -634,3 +634,27 @@ func (h *hyper) killOneContainer(cID string, signal syscall.Signal, all bool) er
 
 	return nil
 }
+
+func (h *hyper) processListContainer(pod Pod, c Container, options ProcessListOptions) (ProcessList, error) {
+	return h.processListOneContainer(pod.id, c.id, options)
+}
+
+func (h *hyper) processListOneContainer(podID, cID string, options ProcessListOptions) (ProcessList, error) {
+	psCmd := hyperstart.PsCommand{
+		Container: cID,
+		Format:    options.Format,
+		PsArgs:    options.Args,
+	}
+
+	proxyCmd := hyperstartProxyCmd{
+		cmd:     hyperstart.PsContainer,
+		message: psCmd,
+	}
+
+	response, err := h.proxy.sendCmd(proxyCmd)
+	if msg, ok := response.([]byte); ok {
+		return msg, err
```

Comment:
This would be clearer as follows (currently it is unclear if it's an error or not :):

```
return msg, nil
```

Reply:
yes I think this should return `return msg, nil` since this is the __right__ case.

Reply:
@sboeuf - Once that's done, is this good to merge?

Reply:
@jodh-intel @devimc Yes after this is cleared up, I think we're good to merge this PR. Please could you implement like this in order to get all this clearer:
```go
response, err := h.proxy.sendCmd(proxyCmd)
if err != nil {
        return nil, err
}
if msg, ok := response.([]byte); !ok {
        return nil, fmt.Errorf("failed to get response message")
}

return msg, nil
```

Reply:
@devimc any progress on this one ?